src/sources/kafka.rs

```diff
@@ -246,7 +246,7 @@ fn default_headers_key() -> OptionalValuePath {
     OptionalValuePath::from(owned_value_path!("headers"))
 }
 
-fn example_auto_offset_reset_values() -> [&'static str; 7] {
+const fn example_auto_offset_reset_values() -> [&'static str; 7] {
```

Comment:
Was this a clippy suggestion?

Reply:
Yep, exactly.

Reply:
👍 I figured, but also thought I remembered seeing one of these merged that wasn't `const`... probably a different return type 🤷